modules/EmergencyMedicine/sql/0_1_0-to-0_2_0_upgrade.sql

```diff
@@ -322,15 +322,55 @@ INSERT INTO `fhir_value_set_systems` (`vs_id`, `system`, `type`,`filter`)
 VALUES ('tests_and_treatments', 'tests_and_treatments', 'All', NULL);
 #EndIf
 
-#IfNotRow fhir_value_sets id x_ray_types
+
+#IfRow2D fhir_value_sets id x_ray_types title Xray Types
+DELETE FROM fhir_value_sets WHERE id ="x_ray_types";
+#EndIf
+
+#IfRow3D fhir_value_set_systems vs_id x_ray_types system x_ray_types type All
+DELETE FROM fhir_value_set_systems WHERE vs_id ="x_ray_types";
+#EndIf
+
+#IfNotRow fhir_value_sets id details_x_ray
 INSERT INTO `fhir_value_sets` (`id`, `title`)
-VALUES ('x_ray_types', 'Xray Types');
+VALUES ('details_x_ray', 'Xray Type');
 #EndIf
 
-#IfNotRow3D fhir_value_set_systems vs_id x_ray_types system x_ray_types type All
+#IfNotRow3D fhir_value_set_systems vs_id details_x_ray system details_x_ray type All
 INSERT INTO `fhir_value_set_systems` (`vs_id`, `system`, `type`,`filter`)
-VALUES ('x_ray_types', 'x_ray_types', 'All', NULL);
+VALUES ('details_x_ray', 'details_x_ray', 'All', NULL);
+#EndIf
+
+#IfRow2D list_options list_id lists option_id x_ray_types
 
+DELETE FROM list_options WHERE list_id="x_ray_types" OR option_id="x_ray_types"
+
+#EndIf
+
+#IfNotRow2D list_options list_id lists option_id details_x_ray
```

Comment:
no reason to change in the list option table

I ask only to change the id and title of the value set

Reply:
"no reason to change in the list option table" - there is I told we work with const - good morning